tests/telemetry_intake/static/config_aggregation_list.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I can see these static files becoming stale pretty quick. Instead of adding these files to system-tests can we update  test_config_telemetry_completeness to use [PyGithub](https://github.com/PyGithub/PyGithub) to clone files from dd-go (using sparse checkout). This way we don't have to document and maintain a shell script and duplicate json files.

Reply:
I'd prefer not to unless this code is in a separate open-source repo, otherwise it'll be broken for anyone without a DD auth'd GH token

see: https://github.com/DataDog/system-tests/pull/3650#discussion_r1880629186